app/views/katello/api/v2/products/base.json.rabl

```diff
@@ -7,13 +7,30 @@ extends 'katello/api/v2/common/org_reference'
 
 attributes :provider_id
 attributes :sync_plan_id
-attributes :sync_status
 attributes :sync_summary
 attributes :gpg_key_id
 attributes :redhat? => :redhat
 
 attributes :available_content => :available_content, :if => params[:include_available_content]
 
+node :sync_status do |product|
```

Comment:
thats my guess of why at least, here is what I am seeing in the rendered json when I do child :sync_status http://fpaste.org/277161/44096471/